src/components/IPScan/Summary/index.js

```diff
@@ -379,7 +365,24 @@ const SummaryIPScanJob = ({
                         Update
                       </button>
```

Comment:
Do you think a cancel button would be useful... in case you don't want to change the title any more?

Reply:
Good point. As we are completely hiding the old title, it is worth adding a button to go back to what it was